src/doc/en/reference/references/index.rst

```diff
@@ -5343,6 +5343,11 @@ REFERENCES:
              Mathematical Physics, Analysis and Geometry 1, 171-191 (1998).
              :doi:`10.1023/A:1009724323513`
 
+.. [PP2010] \C. Paar and J. Pelzl.
+            *Understanding Cryptography*.
```

Comment:
I wasn't sure whether or not to include the subtitle which is "A Textbook for Students and Practitioners".